### PR TITLE
ISSD-2307 [PRM] MDF Claim detail page cards to look clickable

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

**Comment by liferay-customer-delivery** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx [hunk 1]
```diff
@@ -216,6 +215,15 @@ const ActivityClaimPanel = ({
 							</h5>
 						</div>
 					</div>
+
+					{!expanded && activity.selected && (
+						<span className="collapse-icon-closed mt-2">
+							<ClayIcon symbol="angle-down"></ClayIcon>
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx:221]:
```suggestion
							<ClayIcon symbol="angle-down" />
```

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx [hunk 2]
```diff
@@ -216,6 +215,15 @@ const ActivityClaimPanel = ({
...
+						</span>
+					)}
+					<span className="collapse-icon-open mt-2">
+						<ClayIcon symbol="angle-up"></ClayIcon>
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx:225]:
```suggestion
						<ClayIcon symbol="angle-up" />
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx (hunk 1 above)
Comment (abfranca) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx]: Done!

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx (hunk 2 above)
Comment (abfranca) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/components/ActivityClaimPanel/ActivityClaimPanel.tsx]: Done!

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**